modules/machine-user-infra-machines-iso.adoc

```diff
@@ -2,19 +2,46 @@
 //
 // * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
 // * post_installation_configuration/node-tasks.adoc
+ifeval::["{context}" == "multi-architecture-configuration"]
+:multi:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="machine-user-infra-machines-iso_{context}"]
-= Creating more {op-system} machines using an ISO image
+= Creating {op-system} machines using an ISO image
 
 You can create more {op-system-first} compute machines for your bare metal cluster by using an ISO image to create the machines.
 
 .Prerequisites
 
 * Obtain the URL of the Ignition config file for the compute machines for your cluster. You uploaded this file to your HTTP server during installation.
+* You installed the OpenShift CLI (`oc`)
 
 .Procedure
 
+. You can extract the Ignition config file from the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > worker.ign
+----
+
+. Upload the worker.ign Ignition config file you exported from your cluster to your HTTP server. Note the URLs of these files. 
+
+. You can validate that the ignition files are available on the URLs. The following example gets the Ignition config files for the compute node: 
```

Comment:
Would this be better in the form of a verification step? I am not sure if you need to do this before accessing the ISO image. Just a thought because of the way it is worded.